.infrastructure/15_database.tf

```diff
@@ -0,0 +1,28 @@
+# RDS Aurora PostgreSQL Serverless for CMS Strapi
+module "rds" {
```

Comment:
I'd change the name of the module giving it a meaningful one

Reply:
tell me the name because I have to change it later on other PRs too